app/services/response_file_builder/build_response_pdf_file.rb

```diff
@@ -56,127 +63,153 @@ def pdf_fields
       result
     end
 
-    def tri_state_value_for(value, yes: 'yes', no: 'no', off: 'Off')
-      return off if value.nil?
-      value ? yes : no
+    def apply_field(result, field_value, *path)
+      field_def = yaml_data.dig(*path)
+      raise "Field #{path} does not exist in the file #{yaml_file}" unless field_def
+      return if field_def[:field_name] == false
+      if field_def.key?(:select_values)
```

Comment:
The select_values entry in the yaml file states that this field has a single value that are selected from a list.